build.sh

```diff
@@ -84,8 +85,8 @@ apt-get -q update
 apt-get -q -y --purge install ubuntu-standard casper lupin-casper \
   laptop-detect os-prober linux-generic
 
-#
-dpkg -i meilix-metapackage*.deb
+# TODO: Update melix-metapackage to work with Xenial
+#dpkg -i meilix-metapackage*.deb
```

Comment:
Also, it's better to not explicitly use `dpkg` because it doesn't solve dependency issues like this.